Build failure on master

JDK 8 on Travis only:

```
> Task :detekt-gradle-plugin:detekt FAILED
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at io.gitlab.[secure].detekt.cli.Main.parseArguments(Main.kt:61)
	at io.gitlab.[secure].detekt.cli.Main.buildRunner(Main.kt:33)
	at io.gitlab.[secure].detekt.cli.Main.main(Main.kt:17)
Caused by: java.lang.NoClassDefFoundError: org/jetbrains/kotlin/config/JvmTarget
	at io.gitlab.[secure].detekt.cli.CliArgs.<init>(CliArgs.kt:174)
	... 7 more
Caused by: java.lang.ClassNotFoundException: org.jetbrains.kotlin.config.JvmTarget
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 8 more
```

Today I got
```
Caused by: java.lang.NoClassDefFoundError: io/gitlab/arturbosch/detekt/invoke/CustomReportArgument
	at io.gitlab.arturbosch.detekt.Detekt.check(Detekt.kt:235)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:103)
```
for 

```
DetektTaskDslTest
all > io.gitlab.arturbosch.detekt > DetektTaskDslTest
```

just randomly/

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ matrix:
   include:
     - os: linux
       dist: xenial
-      jdk: openjdk8
+      jdk: oraclejdk8
     - os: linux
       dist: bionic
       jdk: openjdk11

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,7 @@ language: java
 matrix:
   include:
     - os: linux
-      dist: xenial
+      dist: trusty
       jdk: oraclejdk8
     - os: linux
       dist: bionic

